pkg/operation/seed/seed.go

```diff
@@ -359,14 +360,27 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 		if err := common.DeleteOldLoggingStack(ctx, k8sSeedClient.Client(), v1beta1constants.GardenNamespace); err != nil {
 			return err
 		}
-		// Read extension provider specific configuration
+
+		// Fetch component specific logging configurations
+		for _, componentFn := range []component.LoggingConfiguration{
+			kubescheduler.LoggingConfiguration,
+		} {
+			parser, filter, err := componentFn()
+			if err != nil {
+				return err
+			}
+
+			filters.WriteString(fmt.Sprintln(filter))
+			parsers.WriteString(fmt.Sprintln(parser))
```

Comment:
Same here as for the monitoring scrape/alert config. Can you move the stuff from the core components to values separate from the `extension` values?